client/src/app/pages/main/nav/nav.component.ts

```diff
@@ -6,18 +6,17 @@ import {Component, ElementRef, HostListener, ViewChild} from '@angular/core';
   styleUrls: ['./nav.component.scss'],
 })
 export class NavComponent {
-
   @HostListener('window:scroll', [])
   onWindowScroll() {
-    if (window.pageYOffset > 200) {
-      this.mainNavClass = 'navbar-shrink';
-    } else {
-      this.mainNavClass = '';
-    }
+    window.pageYOffset > 200 ? this.mainNavClass = 'navbar-shrink' : this.mainNavClass = '';
   }
-  @ViewChild('mainNav', {static: false}) mainNav: ElementRef;
-  mainNavClass: string;
-  constructor() {
-    this.mainNavClass = '';
+
+  @ViewChild('mainNav', {static: false})
+  mainNav: ElementRef;
+  mainNavClass: string = '';
+
+  navbarOpen: boolean = false;
```

Comment:
Why get rid of the constructor for setting these values?

Reply:
I'm... not sure. Not used to using Typescript properly I guess, confusing myself

Reply:
That's okay. We initialize all values in the constructor, along with putting all of variable declarations above it. See [the profile component](https://github.com/momentum-mod/website/blob/master/client/src/app/pages/dashboard/profile/user-profile.component.ts#L35) for an example of what to do here.

Reply:
Thanks, that's helpful. I've read up a bit, hopefully this last commit gets it right.